test/unit/gateways/priority_test.rb

```diff
@@ -6,7 +6,7 @@ def setup
     # run command below to run tests in debug (byebug)
     # byebug -Itest test/unit/gateways/priority_test.rb
 
-    @gateway = PriorityGateway.new(fixtures(:priority))
+    @gateway = PriorityGateway.new(key: 'sandbox_key', secret: 'secret', merchant_id: 'merchant_id')
     @merchant = '1000003310'
```

Comment:
Again, this is unrelated to changes you've made, but something I noticed and would like us to resolve. I think this is a real merchant_id and should probably be removed from the file. Any place that references this variable throughout the test file (I see it on lines  44 and 340) should instead reference `@gateway.options[:merchant_id]` to use the placeholder credentials that are being used for the unit test.

Reply:
Done please check 👍